Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -446,8 +446,12 @@ class BTHTTP: NSObject, URLSessionTaskDelegate {
         let json = try? JSONSerialization.jsonObject(with: data)
         let body = BTJSON(value: json)
         
-        guard let mutationName = body["operationName"].asString() else { return nil }
-        
-        return "mutation \(mutationName)"
+        guard let query = body["query"].asString() else {
+            return nil
+        }
+
+        let queryDiscardHolder = query.replacingOccurrences(of: #"^[^\(]*"#, with: "", options: .regularExpression)
```

Comment:
Open to alternate names for this, essentially this is to hold the parts of the query we want to remove. For example:
```swift
mutation CreateVenmoPaymentContext($input: CreateVenmoPaymentContextInput!) { createVenmoPaymentContext(input: $input) { venmoPaymentContext { id } } }
// now passed as
mutation CreateVenmoPaymentContext
```
Took some messing around with regex in a playground with multiple queries to get the correct syntax

Reply:
take it or leave it - do we wanna also update the function name to `getGraphQLQueryName` to reflect this change